src/dev/build/lib/fs.js

```diff
@@ -47,6 +47,17 @@ export function assertAbsolute(path) {
   }
 }
 
+export function isFileAccessible(path) {
+  assertAbsolute(path);
+
+  try {
+    isFileAccessible(path);
```

Comment:
This is a circular call, I'm kinda shocked that this is `try/catch` able.

Reply:
good catch 👍